src/utils/schema/widgetPropertyUpdaters.ts

```diff
@@ -1,6 +1,127 @@
 import { VernaContextProps } from '../../providers/VernaProvider';
 import VernaJSONSchemaType from '../../types/rjsf';
 import type { Properties } from './updateWidgetProperties';
+import { RJSF_ID_SEPARATOR } from '../../settings';
+
+function getTranslationKey(
+  widgetPath: string[],
+  propertyKey: string,
+  selector: string | undefined,
+) {
+  if (selector) {
+    const translationKeyParts = [widgetPath.shift(), selector, ...widgetPath, propertyKey];
+    return translationKeyParts.join(RJSF_ID_SEPARATOR);
+  } else {
+    return widgetPath.join(RJSF_ID_SEPARATOR).concat(`${RJSF_ID_SEPARATOR}${propertyKey}`);
```

Comment:
Just to be iso with the statement above. From my point of view, it helps to understand the difference between this two cases.

```suggestion
    return [...widgetPath, propertyKey].join(RJSF_ID_SEPARATOR);
```

Reply:
indeed more readable ! :)